### DISCUSSION TITLE
Still getting random crashes

### DISCUSSION THREAD

**Discussion by ghost** [YimMenu/YimMenu#2253] [Q&A]
I still seem to get getting random crashes with the latest release of yim. Don't get me wrong, it is way more stable than before when the integrity checks weren't bypassed but ever since then I've still been getting random crashes. I'm not sure if it could be one of the settings I have enabled or if others are having the same problem as me but as far and I can tell I'm alone on this one qwq. OH and see, I would open an issue for the problem but there's no error log, and I don't wanna be that guy that doesn't upload the log so yeah. If it helps, I seem to only get crashes when I join missions. If I stay in freemode for hours I'm ok but when I start to join missions, I will crash eventually. Sorry if I've typed too much, below you can find a summary of my post generated by ChatGPT.

TLDR: The latest release of YIM is more stable than before, but there are still random crashes, especially when joining missions. The user isn't sure if it's their settings or a common issue. They can't provide an error log, but they want to report the problem. Freemode is fine, but missions cause crashes.

Okay, nevermind about it being ok in freemode. I injected yim, loaded into online, and it says "Received Modder Detection from "<Player name> and I instantly crashed. I'm really not sure what to do other than try to restore my settings to default or cry bc that'll prob do nothing.

**Comment by binded-zz** [comment]
i would say try to turn on all of the logs metrics all of the logs maybe you can see something

**Comment by lonelybud** [comment]
when i played heist today after it got completed I crashed after sometime while loading into a new session. Otherwise i dont see issues.

**Reply by xiaoxiao921** [reply]
I already told you that these offsets don't need to be updated. Maybegreat pr with the game skeleton cb made this entire thing obsolete,  it's just that this code didn't get removed yet

**Reply by ghost** [reply]
So 
What’s the actual cause of these crashes because im having the same issues. Which sucks because i like helping players doing heists

**Reply by xiaoxiao921** [reply]
Well it's definitly AC related due to the lack of meaningful log output. Probably some arxan fuckery but I personally don't know much on how to solve those kind of crashes specifically

**Reply by ghost** [reply]
qwq

**Comment by binded-zz** [comment]
i only really play in freemode  seems like alot of people have problems with missions crashes maybe people making reports should have a option to click i was in freemode i was in mission such such i was starting a mission or  crash error at end of mission 
idk just saying 
maybe it can help sort out where people are crashing with out logs 
theres alot of crashed but no log reports


most of my crashes are from my self overloading  my gta sending to many to fast spawned peds

**Reply by ghost** [reply]
i mean, yeah, details matter because there's certain scripts running during missions compared to freemode with not much going on besides public events and ceo/mc work. Anyway, yeah kinda sad because I like doing missions and helping people. i can't dick around in freemode forever.

**Comment by ghost** [comment]
Turned on nearly all the logging I could, this is all I saw before the game just crashed with no error or anything. 
[had to upload it to pastebin because it wouldn't upload to github idk why](https://pastebin.com/mYJBKugj)
[another crash after doing the casino heist and just typing in chat in freemode](https://pastebin.com/KRC7r8rC)
I've noticed my game seems to crash right after modder detection. Maybe there's something with modder detection that's causing these crashes, idk...

**Reply by xiaoxiao921** [reply]
https://github.com/YimMenu/YimMenu/issues/1269#issuecomment-1515611025

Can you try this? Could contain something useful

**Reply by ghost** [reply]
oooohh, you're so smort

Faulting application name: GTA5.exe, version: 1.0.3028.0, time stamp: 0x650b237f
Faulting module name: unknown, version: 0.0.0.0, time stamp: 0x00000000
Exception code: 0xc0000005
Fault offset: 0x00007ff9031a2374
Faulting process id: 0x50c4
Faulting application start time: 0x01d9fe81905b68f3
Faulting application path: O:\Epic Games\GTAV\GTA5.exe
Faulting module path: unknown
Report Id: 0d38371b-fa1d-487a-9d3b-fa501215bde5
Faulting package full name: 
Faulting package-relative application ID: 

another crash
Faulting application name: GTA5.exe, version: 1.0.3028.0, time stamp: 0x650b237f
Faulting module name: ucrtbase.dll, version: 10.0.22000.1, time stamp: 0x00e78ce9
Exception code: 0xc0000409
Fault offset: 0x000000000007dd7e
Faulting process id: 0x6eb0
Faulting application start time: 0x01d9fe8125318e1d
Faulting application path: O:\Epic Games\GTAV\GTA5.exe
Faulting module path: C:\WINDOWS\System32\ucrtbase.dll
Report Id: 75eac988-a758-4bbc-9076-3b028e1bc5b0
Faulting package full name: 
Faulting package-relative application ID: 

another
Faulting application name: GTA5.exe, version: 1.0.3028.0, time stamp: 0x650b237f
Faulting module name: unknown, version: 0.0.0.0, time stamp: 0x00000000
Exception code: 0xc0000005
Fault offset: 0x00007ff9031a2374
Faulting process id: 0xfec
Faulting application start time: 0x01d9fe79ac71904a
Faulting application path: O:\Epic Games\GTAV\GTA5.exe
Faulting module path: unknown
Report Id: 276aa656-f2ce-4764-8129-e980d7e01b1e
Faulting package full name: 
Faulting package-relative application ID: 

and another
Faulting application name: GTA5.exe, version: 1.0.3028.0, time stamp: 0x650b237f
Faulting module name: unknown, version: 0.0.0.0, time stamp: 0x00000000
Exception code: 0xc0000005
Fault offset: 0x00007ff9031a2374
Faulting process id: 0x8138
Faulting application start time: 0x01d9fe500e42f155
Faulting application path: O:\Epic Games\GTAV\GTA5.exe
Faulting module path: unknown
Report Id: 7327743e-9424-471c-8677-2b4f45aad155
Faulting package full name: 
Faulting package-relative application ID:

**Reply by xiaoxiao921** [reply]
Most likely arxan cause the stack is fucked, pinging just in case you have some ideas @maybegreat48

**Reply by thelaalkutro** [reply]
Did anyone ever figure out the issue that caused the heist or mission to crash? I got Yim just last week and it works fine under the freemode just when I join any mission or heist it crashes or puts me back in the freemode.
